workflow/react-project/src/components/generic/Button/index.jsx

```diff
@@ -0,0 +1,9 @@
+import './style.less';
+
+const Button = ({ children, style, onClick }) => (
+  <button type="button" style={style} onClick={onClick} className="button">
```

Comment:
if you want to style common component somehow in different places, you don't need to write inline styles for it. 
You can also pass `className` and receive it in you component. 
For ex: 
  `const Button = ({ children, className, onClick }) =>  <button type="button" onClick={onClick} className=``button ${className}``>`
  
  But if we don't pass anything in classNames to the Button, we will have classes like: `button undefined`
  
  So there're useful library for working with classes ([classnames](https://www.npmjs.com/package/classnames) and [clsx](https://www.npmjs.com/package/clsx)), you can install one of them and try to use.